files/fr/glossary/first_cpu_idle/index.html

```diff
@@ -0,0 +1,6 @@
+---
+title: Première inactivité CPU
+slug: Glossary/First_CPU_idle
+translation_of: 'Glossary/First_CPU_idle'
+---
+<p><strong>La première inactivité CPU</strong> (« <i>First CPU Idle</i> » en anglais) mesure quand une page est minimalement interactive, ou quand la fenêtre est suffisamment silencieuse pour gérer les entrées de l'utilisateur. Il s'agit d'une mesure de performance web non standard de Google. Généralement, cela se produit lorsque la plupart des éléments d'interface utilisateur (UI) visibles, mais pas nécessairement tous, sont interactifs et que l'interface utilisateur répond, en moyenne, à la plupart des entrées de l'utilisateur dans les 50 ms. Il est également connu sous le nom de <a href="/fr/docs/Glossary/First_interactive">Premier interactif</a>.</p>
```

Comment:
Hmm, I'm not sure this quite conveys the performance thing in French. Personally, I'd choose something a bit more verbose like « Premier instant d'interactivité"

Reply:
@ManuSquall let me know what you think of this and let's merge afterwards.

Reply:
Hello @SphinxKnight, I'm ok with that, guess it is more accurate in french, thank you!!